sites/all/modules/unl_cas/unl_cas.module

```diff
@@ -125,6 +125,9 @@ function unl_cas_validate_ticket() {
   if ($auth) {
     $username = $cas->getUsername();
     $user = unl_cas_import_user($username);
+    if (!$user) {
+      drupal_set_message('An error occured importing the user.', 'error');
```

Comment:
Probably. But really this should never* be a problem because if $auth is true then the import user function should return a user.  More robustness could be added in case LDAP is offline and it fails.... but nothing existing before so I thought I'd throw something in to get started.